app/scripts/metamask-controller.js

```diff
@@ -4674,16 +4680,11 @@ export default class MetamaskController extends EventEmitter {
       trezorKeyring.dispose();
     }
 
-    const [ledgerKeyring] = this.coreKeyringController.getKeyringsByType(
```

Comment:
Is this not necessary anymore? Does something else do this instead?

Reply:
`eth-keyring-controller@13` is responsible for disposing ledger or trezor keyring instead of the clients: https://github.com/MetaMask/KeyringController/blob/12e3a2017a9f9b76e10497d93746deeefe3e190b/src/KeyringController.ts#L990

But as the `eth-trezor-keyring` version that we are using in the extension still has a `dispose` method instead of `destroy` (we renamed this in the latest version), we have to keep this direct call to `trezorKeyring.dispose();` for now

Reply:
Ah, thank you, that makes sense.